spec/index.bs

```diff
@@ -6187,18 +6204,27 @@ attachments used by this encoder.
 
     : <dfn>writeTimestamp(querySet, queryIndex)</dfn>
     ::
+        Writes a timestamp value into |querySet| when all previous commands have completed executing.
 
         <div algorithm="GPURenderPassEncoder.writeTimestamp">
-            **Called on:** {{GPURenderPassEncoder}} this.
+            **Called on:** {{GPURenderPassEncoder}} |this|.
 
             **Arguments:**
             <pre class=argumentdef for="GPURenderPassEncoder/writeTimestamp(querySet, queryIndex)">
```

Comment:
did we consider a partial interface having this stuff, so that we can specify it once instead of repeating ourselves for every type of pass and the command encoder?

Reply:
I don't think we have closely considered it (a `mixin`, I think you mean). I guess it depends on whether we expect anything in the algorithm to differ between them, but if not, we should probably try to do it.